index.html

```diff
@@ -10711,12 +10711,21 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
-				<p>Unlike elements referenced by <code>aria-describedby</code>, the element referenced by <code>aria-details</code> is not used in either the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_name">Name Computation</a> or the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description">Description Computation</a> as defined in the Accessible Name and Description specification [[ACCNAME-1.1]]. Thus, the content of an element referenced by <code>aria-details</code> is not flattened to a string when presented to assistive technology users. This makes <code>aria-details</code> particularly useful when converting the information to a string would cause a loss of information or make the extended description more difficult to understand.</p>
+				<p>Identifies one or more elements that provide additional, extended information about the <a>object</a>.</p>
+				<p>The additional information can be:</p>
+				<ul>
+					<li>A description or general annotation (this is the default). See related <pref>aria-describedby</pref>.</li>
+					<li>A <rref>comment</rref> &mdash; in this case the element referred to SHOULD use the <rref>comment</rref> role.</li>
+					<li>A <rref>definition</rref> &mdash; in this case the referring element SHOULD use the <rref>term</rref> role and the element referred to SHOULD use the <rref>definition</rref> role.</li>
+					<li>A footnote &mdash; in this case the element referred to SHOULD use the <code>doc-footnote</code> role from [[DPUB-ARIA]].</li>
+					<li>An endnote &mdash; in this case the element referred to SHOULD use the <code>doc-endnote</code> role from [[DPUB-ARIA]].</li>
+				</ul>
+
+				<p>The <code>aria-details</code> attribute references elements that provide more detailed information than would normally be provided by <pref>aria-describedby</pref>. It enables <a>assistive technologies</a> to make users aware of the availability of extended information as well as navigate to it. Authors SHOULD ensure elements referenced by <code>aria-details</code> are visible to all users.</p>
+				<p>Unlike elements referenced by <code>aria-describedby</code>, elements referenced by <code>aria-details</code> are not used in either the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_name">Name Computation</a> or the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description">Description Computation</a> as defined in the Accessible Name and Description specification [[ACCNAME-1.1]]. Thus, the content of elements referenced by <code>aria-details</code> are not flattened to a string when presented to assistive technology users. This makes <code>aria-details</code> particularly useful when converting the information to a string would cause a loss of information or make the extended description more difficult to understand.</p>
```

Comment:
```suggestion
				<p>Unlike elements referenced by <code>aria-describedby</code>, elements referenced by <code>aria-details</code> are not used in either the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_name">Name Computation</a> or the Accessible <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_description">Description Computation</a> as defined in the Accessible Name and Description specification [[ACCNAME-1.1]]. Thus, the content of elements referenced by <code>aria-details</code> are not flattened to a string when presented to assistive technology users. This makes <code>aria-details</code> particularly useful when converting the information to a string would cause a loss of information or make the extended information more difficult to understand.</p>
```